R/catchmentArea.R

```diff
@@ -98,14 +98,14 @@ calc_catchment <- function(
     targetintersectd <- sp::SpatialPolygonsDataFrame(targetintersectd,
                                                      data=if(length(calccols) == 1){
                                                        setNames(
-                                                         data.frame(x=sum(targetintersect@data[,calccols])),
+                                                         data.frame(x=sum(targetintersect@data[,calccols], na.rm = TRUE)),
                                                          calccols
                                                        )}else{
-                                                         data.frame(as.list(colSums(targetintersect@data[,calccols])))
+                                                         data.frame(as.list(colSums(targetintersect@data[,calccols],na.rm = TRUE)))
```

Comment:
Would be a little happier if there were spaces after the commas, i.e. (using Ctl-Shift-A in RStudio):

```
data.frame(as.list(colSums(targetintersect@data[, calccols], na.rm = TRUE))) 
```

Reply:
I'll add the spaces. Bad habit of mine to revert to no spaces when I have compact code but I also prefer the spaces to be in there.